py3status/core.py

```diff
@@ -596,6 +596,9 @@ def notify_user(self, msg, level='error', rate_limit=None, module_name=''):
         msg_hash = hash(u'{}#{}#{}'.format(module_name, limit_key, msg))
         if msg_hash in self.notified_messages:
             return
+        elif module_name:
+            x = 'Module `%s` sent a notification. "%s"' % (module_name, msg)
+            self.log(x, level)
         else:
             self.log(msg, level)
             self.notified_messages.add(msg_hash)
```

Comment:
you have changed the logic here.
```
self.notified_messages.add(msg_hash)
```
Is very important, if you unindent this line then we will be good.

Reply:
Ping. All good.